RedJanvier/eshuri-bank-fe
# ft: Apply for Card Page

### What this PR does
It contains a reusable js sidebar which is in `src/utils/loadSideBar.js`
It contains a javascript file to load the content `src/utils/loadPageContent.js`
It contains a folder called `components` which will contain all our javascript pages (e.g `DashBoard.js` , `ApplyForCard.js`)

I used a function based approach to define the components and pages to be used in the entire application since it is small and light weight and we are not using javascript libraries

### How to test the frontend routes
After running `npm run dev` go in the file called loadPageContent.js located  in `src/utils/loadPageContent.js` and see defined routes

- Netlify link and the visit `/apply-card`
- You can visit `http://localhost:5173/apply-card` to view my changes 


#### Screenshots
![Screenshot 2024-12-17 185840](https://github.com/user-attachments/assets/8975063e-e3c0-48e7-8f95-503e37a9f12c)


### Detailed explanations

- `src/components/**` : This folder will contain all our Page javascript components
- `src/utils/loadSideBar.js` : This file dynamically generates the sidebar 
- `src/utils/loadPageContent.js`: defining routes
